sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/metadata/datasource/dialect/MariaDBDataSourceMetaData.java

```diff
@@ -41,6 +44,8 @@
 
     private final String schemaName;
 
+    private final String catalog;
+
     private final Pattern pattern = Pattern.compile("jdbc:(mysql|mariadb)(:replication|:failover|:sequential|:aurora)?:(\\w*:)?//([\\w\\-\\.]+):?([0-9]*)/([\\w\\-]+);?\\S*", Pattern.CASE_INSENSITIVE);
 
     public MariaDBDataSourceMetaData(final String url) {
```

Comment:
I think it is not necessary after `MariaDBDataSourceMetaData(final DataSourceInfo dataSourceInfo)` is added.